### PR TITLE
Adding other CD4s

### PR DESCRIPTION
**Pull request by Marymary-dev**

### CODE REVIEW

File: Scripts/NDWH/C&T FACT TABLES/load_FactCD4.sql [hunk 1]
```diff
@@ -25,25 +25,48 @@ with MFL_partner_agency_combination as (
...
-		bCD4 as BaselineCD4,
-		bCD4Date as BaselineCD4Date,
+		BaselineCD4s.TestResult as BaselineCD4,
+		BaselineCD4s.OrderedbyDate as BaselineCD4Date,
 		LatestCD4s.OrderedbyDate as LastCD4Date,
         Case When LatestCD4s.TestName='CD4 Count'Then LatestCD4s.TestResult Else Null End as LastCD4,
         Case When LatestCD4s.TestName='CD4 Percentage' Then LatestCD4s.TestResult Else Null End as LastCD4Percentage,
-		datediff(yy, patient.DOB, last_encounter.LastEncounterDate) as AgeLastVisit
+		datediff(yy, patient.DOB, last_encounter.LastEncounterDate) as AgeLastVisit,
+        OtherCD4s.TestResult as OtherCD4s,
```
Comment (nobert-mumo) [inline Scripts/NDWH/C&T FACT TABLES/load_FactCD4.sql:60]: @Marymary-dev just curious why the name `OtherCD4s` ?

Comment (Marymary-dev) [reply Scripts/NDWH/C&T FACT TABLES/load_FactCD4.sql:60]: @nobert-mumo  we adopted the name just as it is in the mock up ; 

image

Comment (nobert-mumo) [reply Scripts/NDWH/C&T FACT TABLES/load_FactCD4.sql:60]: @Marymary-dev ah okay, am just afraid the name could be a bit vague coz I see the logic is to pick the 2nd earliest CD4

Comment (Marymary-dev) [reply Scripts/NDWH/C&T FACT TABLES/load_FactCD4.sql:60]: @nobert-mumo  alright renaming it to SecondCD4

Comment (Marymary-dev) [reply Scripts/NDWH/C&T FACT TABLES/load_FactCD4.sql]: @nobert-mumo  requesting re-review on this

**Review by nthusi-codes — APPROVED** [review]

**Pull request merged**